tests/next_tests/integration_tests/feature_tests/iterator_tests/test_builtins.py

```diff
@@ -180,13 +178,14 @@ def test_arithmetic_and_logical_functors_gtfn(builtin, inputs, expected):
     inps = field_maker(*array_maker(*inputs))
     out = field_maker((np.zeros_like(*array_maker(expected))))[0]
 
-    gtfn_executor = run_gtfn.executor
     gtfn_without_transforms = dataclasses.replace(
-        gtfn_executor,
-        otf_workflow=gtfn_executor.otf_workflow.replace(
-            translation=gtfn_executor.otf_workflow.translation.replace(enable_itir_transforms=False)
-        ),
-    )  # avoid inlining the function
+        run_gtfn,
+        executor=dataclasses.replace(
+            run_gtfn.executor,
```

Comment:
```suggestion
        executor=run_gtfn.executor.replace(
```

Just a nitpick: The `dataclasses.replace` was only necessary for the `ModularExecutor` instance, because that was not a replaceable workflow. (Admittedly small) benefits to using `.replace` would be brevity and consistency.